openfl/protocols/director.proto

```diff
@@ -185,13 +185,25 @@ message GetEnvoysResponse {
     repeated EnvoyInfo envoy_infos = 1;
 }
 
+message SetExperimentFailedRequest {
+    string experiment_name = 1;
+    string collaborator_name = 2;
+    uint32 error_code = 3;
```

Comment:
Do we really need error code for now? It looks like it always `1`

Reply:
yes, you are right, it always 1, but we already have one error type and its description, and code to handle them (write specific log message at least). And view, that there are another error type, and we should handle them in the future. It's the groundwork for the future.